mmp_to_musicxml/converter.py

```diff
@@ -203,16 +203,21 @@ class MMP_MusicXML_Converter:
 	
 	SPECIFIED_KEY_SIGNATURE = None
 	
-	def __init__(self, check_notes=False, key_signature=None):
+	def __init__ ( self, key_signature = None, params = None ):
 		logging.basicConfig(level=logging.DEBUG)
-		
-		if check_notes:
+
+		self.opts = params['opts']
+		self.minor = params['minor']
+
+		if self.opts.check:
 			logging.debug("note checking is on")
 			self.NOTE_CHECKER = NoteChecker()
 			
 		if key_signature:
 			if key_signature in self.FIFTHS:
 				logging.debug(f"adjusting notes per key signature: {key_signature}")
+				if self.minor:
+					logging.debug( 'minor mode [%s]' % self.opts.key )
```

Comment:
sorry I'm being annoying about the formatting (I probably should set up some linting at some point) - mind if we remove the spaces after and before the parentheses?

any particular reason for choosing this way of string formatting vs f-strings? e.g. `f"minor mode {self.opts.key}"`

Reply:
I don't mind it at all. I'm going to drop this pull request, reset the formatting to your taste, and submit it again soon.

Reply:
sounds good, thank you! and awesome work btw.